importer/SeShipSv.py

```diff
@@ -118,6 +119,13 @@ def set_monuments_all_id(self):
         """
         self.monuments_all_id = self.signal
 
+    def set_descriptions(self):
+        """Set default descriptions."""
+        DESC_BASES = {"sv": "K-märkt fartyg",
```

Comment:
If you are not using this as a global then I would use a lower case variable name